app/controllers/breadcrumb_helpers.rb

```diff
@@ -3,7 +3,12 @@
 #
 # See https://github.com/weppos/breadcrumbs_on_rails
 module BreadcrumbHelpers # Included in ApplicationController
+  def add_organization_breadcrumb
+    add_breadcrumb "Organization #{@organization.name}", organization_path(@organization)
```

Comment:
This could have `organization = @organization || @course.organization` which might help avoid some `@organization = @course.organization` statements elsewhere. But do this only if you agree it's an improvement, I'm not entirely sure.